### PR TITLE
nl - Added index to images and show them in descending order

### LINKED ISSUE #3: Image Index
#### As a user I want the homepage to display all saved images.

Acceptance criteria:
- [x] New images that are added show up on the homepage.
- [x] These images are persisted if the browser is closed or even if the
  server is restarted.
- [x] Images are not displayed wider than 400px.

Dependencies:
- Save Image Link

### PR DESCRIPTION
**Pull request by nicklyz**
Closes #3 

Added index to images, show them in descending order in home page.

### CODE REVIEW

File: app/views/home/index.html.erb [hunk 1]
```diff
@@ -12,7 +12,11 @@
     <%= content_tag :div, msg, :class => name %>
 <% end -%>
 
+<ul>
 <% @images.each do |img| %>
```
Comment (weixing2014) [inline app/views/home/index.html.erb]: Might need two more space here.

File: app/models/image.rb [hunk 2]
```diff
@@ -1,2 +1,3 @@
 class Image < ActiveRecord::Base
+  scope :ordered, -> { order(created_at: :desc) }
```
Comment (weixing2014) [inline app/models/image.rb]: You might want to give a more readable name to `scope`. I might name it `:order_by_created_at`

File: app/views/home/index.html.erb [hunk 3]
```diff
@@ -12,7 +12,11 @@
...
-    <p><%= link_to "#{image_tag(img.link, :width => '400px' )}".html_safe, img.link %></p>
+    <li>
+      <%= link_to "#{image_tag(img.link, :width => '400px' )}".html_safe, img.link %>
```
Comment (weixing2014) [inline app/views/home/index.html.erb]: It seems you don't need to use `#{}` here?

Comment (nicklyz) [inline app/views/home/index.html.erb]: > It seems you don't need to use #{} here?

right, I don't know why I added it.

**Pull request merged**